internal/utils/csi.go

```diff
@@ -359,7 +375,9 @@ var HandleVolumeInuseErrorContainerArg = "--handle-volume-inuse-error=false"
 var PodUidContainerArg = fmt.Sprintf("--pod-uid=$(%s)", PodUidEnvVar.Name)
 var PodContainerArg = fmt.Sprintf("--pod=$(%s)", PodNameEnvVar.Name)
 var NamespaceContainerArg = fmt.Sprintf("--namespace=$(%s)", PodNamespaceEnvVar.Name)
-var ControllerPortContainerArg = fmt.Sprintf("--controller-port=%d", CsiAddonsContainerPort.ContainerPort)
+var ControllerPluginPortContainerArg = ContainerPortArg(ControllerPluginCsiAddonsContainerPort)
+var NodePluginPortContainerArg = ContainerPortArg(NodePluginCsiAddonsContainerPort)
```

Comment:
no need for this one as we can avoid extra vars

Reply:
@Madhu-1  wrote:
> no need for this one as we can avoid extra vars

agreed. removed.